Formula/op25.rb

```diff
@@ -0,0 +1,64 @@
+class Op25 < Formula
+  desc "Software-defined analyzer for APCO P25 signals"
+  homepage "https://osmocom.org/projects/op25/wiki"
+  url "git://op25.osmocom.org/op25.git"
+
+  depends_on "cmake" => :build
+  depends_on "cppunit" => :build
+  depends_on "swig" => :build
+  depends_on "gnuradio"
+  depends_on "gr-osmosdr"
+  depends_on "itpp"
+
+  def install
+    # TODO: apply patch if gnuradio >= 3.8?
+
+    args = std_cmake_args + %w[
+      -Wno-dev
```

Comment:
Suppresses warnings intended for project development, but they're only visible when installing with `--verbose`, should this just be removed?